test/tchannel_bridge.js

```diff
@@ -122,7 +122,6 @@ describe ('test tchannel span bridge', () => {
                     let serverSpanTags = {};
                     serverSpanTags[opentracing.Tags.PEER_SERVICE] = 'echo';
                     serverSpanTags[opentracing.Tags.SPAN_KIND] = opentracing.Tags.SPAN_KIND_RPC_SERVER;
-                    serverSpanTags[opentracing.Tags.PEER_HOST_IPV4] = ((127 << 24) | 1);
```

Comment:
Actually, what happened is that the tracer added the peer host tag after the tchannel bridge added it. The tchannel bridge ip was set to 127.0.0.1 but the tracer ip is dynamic.